mappings/net/minecraft/data/server/advancement/HusbandryAdvancementTabGenerator.mapping

```diff
@@ -7,15 +7,25 @@ CLASS net/minecraft/unmapped/C_vfenxpjt net/minecraft/data/server/advancement/Hu
 	FIELD f_zngbqyui FISH_BUCKETS [Lnet/minecraft/unmapped/C_vorddnax;
 	METHOD m_bkajmxpf appendBreedAllEntitiesCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;Ljava/util/stream/Stream;Ljava/util/stream/Stream;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
 		ARG 0 task
+		ARG 1 instantBreedingAnimals
+		ARG 2 eggLayingAnimals
 	METHOD m_fehmbxhu appendEatAllFoodCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
+		ARG 0 task
 		ARG 1 task
 	METHOD m_fqhhoyxq appendTameAllCatVariantsCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
+		ARG 0 task
 		ARG 1 task
 	METHOD m_mjrojbaf appendBucketAllFishCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
+		ARG 0 task
 		ARG 1 task
 	METHOD m_msnwtfou appendLeashAllFrogTypesCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
+		ARG 0 task
 		ARG 1 task
 	METHOD m_ropwgspa appendCatchAllFishCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
+		ARG 0 task
 		ARG 1 task
```

Comment:
these appear to all have two `task` args

Reply:
in enigma all that's visible is the first one, I'll remove the arg 1 from all of them